ksql-serde/src/test/java/io/confluent/ksql/serde/connect/KsqlConnectDeserializerTest.java

```diff
@@ -0,0 +1,90 @@
+package io.confluent.ksql.serde.connect;
+
+import static org.hamcrest.Matchers.sameInstance;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.fail;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import io.confluent.common.logging.StructuredLogger;
+import io.confluent.ksql.GenericRow;
+import io.confluent.ksql.processing.log.ProcessingLogMessageFactory;
+import io.confluent.ksql.serde.SerdeTestUtils;
+import java.util.Optional;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaAndValue;
+import org.apache.kafka.connect.storage.Converter;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+public class KsqlConnectDeserializerTest {
+  private static final String TOPIC = "topic";
+  private static final byte[] BYTES = "bizbazboz".getBytes();
+
+  @Mock
+  private Converter converter;
+  @Mock
+  private DataTranslator dataTranslator;
+  @Mock
+  private StructuredLogger recordLogger;
+  @Mock
+  private Schema schema;
+  @Mock
+  private Object value;
+  @Mock
+  private GenericRow genericRow;
+
+  private KsqlConnectDeserializer connectDeserializer;
+
+  @Rule
+  public final MockitoRule mockitoRule = MockitoJUnit.rule();
```

Comment:
Out of curiosity, why the preference for MockitoRule over MockitoJUnitRunner? Are they different?

Reply:
Not sure about which is preferable. They both seem to do the same stuff.